test/FunctionalTests/Microsoft.OData.Core.Tests/JsonLight/ODataJsonLightDeltaReaderTests.cs

```diff
@@ -974,16 +1027,23 @@ public void ReadEmptyDeltaResourceSetIn41DeletedEntry()
             Assert.NotNull(nestedResourceInfo);
             nestedResourceInfo.Name.Should().Be("FavouriteProducts");
             Assert.NotNull(nestedDeltaResourceSet);
-            nestedDeltaResourceSet.Count.Should().Be(2);
-            nestedDeltaResourceSet.NextPageLink.Should().Be("http://host/service/Customers?$skipToken=5");
+            if (isResponse)
+            {
+                nestedDeltaResourceSet.Count.Should().Be(2);
+                nestedDeltaResourceSet.NextPageLink.Should().Be("http://host/service/Customers?$skipToken=5");
+            }

```

Comment:
maybe add:
else
{
      nestedDeltaResourceSet.Count.Should().Null();
}

Reply:
If count was set then it should be non-null. The only reason it's null is because I didn't set it for the request.

---
In reply to: [256644006](https://github.com/OData/odata.net/pull/1404#discussion_r256644006) [](ancestors = 256644006)